package.json

```
@@ -33,6 +33,16 @@
     "test:parser:performance": "bun run tests/parser/test-runner.ts performance",
     "test:parser:quick": "bun test tests/parser/memory.test.ts tests/parser/node-processor.test.ts tests/parser/system-node-filter.test.ts",
     "test:generate-data": "bun run tests/parser/test-data-generator.ts",
+    "test:database": "bun run tests/database/test-runner.ts",
```

Biome (2.1.2)

[error] The key test:database was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)